sale_elaboration/security/ir.model.access.csv

```diff
@@ -0,0 +1,2 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+access_elaboration_user,access_elaboration_user,model_elaboration,base.group_user,1,1,1,1
```

Comment:
I don't think elaboration should be created by all employees. They should be created by sales managers, and readable by all employees. Check also what happens with portal/public users.